kvbc/src/ReplicaImp.cpp

```diff
@@ -334,7 +334,15 @@ void ReplicaImp::insertBlockInternal(BlockId blockId, Sliver block) {
 }
 
 Sliver ReplicaImp::getBlockInternal(BlockId blockId) const {
-  assert(blockId <= m_lastBlock);
+  if (blockId > m_lastBlock) {
+    LOG_ERROR(logger,
```

Comment:
This should never happen in the old desgin of the KVBC. I suggest reconsidering the issue

Reply:
Do you mean we should keep the assert, instead of returning an empty sliver?

Reply:
No. I only suggest thinking about it again. According to the original design, this should never happen (unless the code contains a bug).
However, I may be missing something (I am not aware of the updates made in recent months).